src/type-extensions.ts

```diff
@@ -179,6 +180,8 @@ declare module "hardhat/types/runtime" {
             getTransaction: (txHash: string) => Promise<Transaction>;
 
             getTransactionReceipt: (txHash: string) => Promise<TransactionReceipt>;
+
+            getBlock: (identifier: BlockIdentifier) => Promise<Block>;
```

Comment:
Add JSdoc to the new function to explain how it works

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/97/commits/70a0fe44cce13506ae7adb25ceef36167730cde2